src/prefect/utilities/notifications.py

```diff
@@ -152,6 +157,13 @@ def slack_message_formatter(
     return data
 
 
+def jira_message_formatter(tracked_obj: TrackedObjectType, state: "prefect.engine.state.State") -> str:
+    time = datetime.now()
```

Comment:
This call will create a timestamp in the local timezone; you might want to consider using `datetime.utcnow()` instead